stablehlo/reference/Ops.cpp

```diff
@@ -541,6 +546,21 @@ Tensor evalLogisticOp(const Tensor &operand, TensorType resultType) {
   return result;
 }
 
+Tensor evalMapOp(ArrayRef<Tensor> inputs, Region &computation, Scope &scope,
+                 TensorType resultType) {
+  Tensor result(resultType);
+  for (auto resultIt = inputs[0].index_begin();
+       resultIt != inputs[0].index_end(); ++resultIt) {
+    SmallVector<Tensor> args;
+    for (size_t i = 0; i < inputs.size(); ++i)
+      args.push_back(
+          Tensor(RankedTensorType::get({}, inputs[i].get(*resultIt).getType()),
```

Comment:
Can we use `computation.getArgument(i).getType()` instead? Shorter to type and doesn't require creating types, i.e. avoids potential logic errors.